tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -51,13 +51,7 @@
 # @result namespacestr [String] Returns 'sudo ip netns exec vrf <cmd>'
 # command string for 'cisco' platform.
 def get_namespace_cmd(host, cmdstr, options)
-  case host['platform']
-  when /cisco/
-    agentvrf = options[:HOSTS][host.to_s.to_sym]['vrf']
-    return "sudo ip netns exec #{agentvrf} " + cmdstr
-  else
-    return cmdstr
-  end
+  return cmdstr
```

Comment:
Using the vrf context will all be handled by beaker.  Beaker will store the value of the vrf and use it when executing commands on the switch.  There really won't even be a need for this function once the next version of Beaker is released.